Directory.Build.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+
+    <PropertyGroup Condition=" '$(MSBuildProjectExtension)' == '.csproj' ">
+        <LangVersion>8.0</LangVersion>
+        <IsEnableNullable Condition=" '$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND $([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), '3.0')) ">True</IsEnableNullable>
+        <IsEnableNullable Condition=" '$(TargetFrameworkIdentifier)' == '.NETStandard' AND $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '2.1')) ">True</IsEnableNullable>
+        <Nullable Condition=" '$(IsEnableNullable)' == 'True' ">enable</Nullable>
+        <Nullable Condition=" '$(IsEnableNullable)' != 'True' ">annotations</Nullable>
+    </PropertyGroup>
```

Comment:
I'm sorry, but this doesn't seem to really work.

Here's how I'm checking it:
1. Change `Atom WithPreviousAtom(DummyAtom? previousAtom);` with `Atom WithPreviousAtom(DummyAtom previousAtom);` in `IRow.cs`
2. `cd src/WpfMath && dotnet build`

**Expected result:** build should generate a couple of warnings.
**Actual result:** no warnings.

If I put back the lines with `<Nullable>` in `WpfMath.csproj` and comment-out the lines in `Directory.Build.props`, then it works.

Reply:
Hmmm, I don’t know why it don’t evaluating in the end

Used the new api https://github.com/dotnet/msbuild/issues/5171